frontend/src/Components/Chart/CartesianCharts/utils/types.ts

```diff
@@ -0,0 +1,49 @@
+export type CartesianChartsData = {
+  label: string;
+  value: number;
+};
+
+export type CartesianChartProps = {
+  chartTitle: string;
+  data: CartesianChartsData[];
+  size: 'small' | 'medium' | 'large' | 'xlarge';
+  xAxisLegend: string;
+  yAxisLegend: string;
+  yLabelCount: number;
+  hasXDirLines?: boolean;
+  hasYDirLines?: boolean;
+  splitYLabel?: [number, number];
+  splitXLabel?: [number, number];
+};
+
+export type CartesianChartsSize = {
+  cWith: number;
+  cHeight: number;
```

Comment:
Her må hjernen jobbe ekstra for å skjønne hva `c` er. Syns det er mye tydeligere å bare ha `width`/`height`

```suggestion
  width: number;
  height: number;
```